script.js

```diff
@@ -16,3 +16,201 @@ document.querySelectorAll('.nav_link').forEach((n) => n.addEventListener('click'
   navBranding.classList.remove('active');
   navContainer.classList.remove('active');
 }));
+
+// multistories project
+
+const multiPost = {
+  id: '_1',
+  heading: 'My Recent Works',
+  name: 'Multi-Post Stories',
+  desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+  featuredImg: 'image folder/works.png',
+  technologies: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+  liveLink: '#',
+  sourceLink: '#',
+};
+
+const stories = document.querySelector('.multi_stories');
+stories.innerHTML += `<div class="recent_work_div flex">
+ <h2 class="recent_work_header">${multiPost.heading}</h2>
+ <img class="image_line" src="image folder/recent_work_line.png" alt="line">
+ </div>
+
+ <div class="recent_work_content">
+ <img class="img_placeholder" src="${multiPost.featuredImg}" alt="multipost image">
+ <div class="placeholder_content">
+ <h3 class="multi_story">${multiPost.name}</h3>
+  <p class="recent_work_text">${multiPost.desc}</p>
+   <ul class="recent_work_language">
+ <li class="work_language flex">${multiPost.technologies[0]}</li>
+ <li class="work_language flex">${multiPost.technologies[1]}</li>
+ <li class="work_language flex">${multiPost.technologies[2]}</li>
+ <li class="work_language flex">${multiPost.technologies[3]}</li>
+  </ul>
+  <button data-modal-target="#modal" type="submit" class="see_prjt_buttn">See Project</button>
```

Comment:
You did a great job together 👍  But let's follow the template and add 'See Project' button to the featured project. It also need popup.
<img width="458" alt="Screenshot 2023-04-13 at 8 54 52 PM" src="https://user-images.githubusercontent.com/25887253/231847205-5da0716b-0cb1-4d0e-8021-dfbf449e3556.png">
<img width="458" alt="Screenshot 2023-04-13 at 8 55 04 PM" src="https://user-images.githubusercontent.com/25887253/231847221-6695e58a-f584-4863-a1ad-0422938befd8.png">